docs/tasks/stateful-sets/deleting-pods.md

```diff
@@ -0,0 +1,70 @@
+---
+assignees:
+- bprashanth
+- erictune
+- foxish
+- smarterclayton
+
+---
+
+{% capture overview %}
+This page shows how to delete pods which are part of a stateful set, and explains the various considerations when doing so.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you are not using Stateful Sets, you can skip this task now. 
+* This is a fairly advanced task and has the potential to violate some properties provided by Stateful Sets.
+* If you are attempting this, you must be well aware of the various considerations enumerated below, and have specific domain knowledge of the application(s) running using Stateful Set.
```

Comment:
"Before proceeding, make yourself familiar with the considerations enumerated below,"